milli/Cargo.toml

```diff
@@ -14,7 +14,8 @@ flate2 = "1.0.20"
 fst = "0.4.5"
 fxhash = "0.2.1"
 grenad = { git = "https://github.com/Kerollmops/grenad.git", rev = "3adcb26" }
-heed = { version = "0.10.6", default-features = false, features = ["lmdb", "sync-read-txn"] }
+# heed = { version = "0.10.6", default-features = false, features = ["lmdb", "sync-read-txn"] }
+heed = { path = "../../heed/heed", default-features = false, features = ["lmdb", "sync-read-txn"] }
```

Comment:
Whoops, what's that?

Reply:
I pushed a wrong commit but now it's fixed :shushing_face: